core/src/main/scala/pureconfig/ConfigCursor.scala

```diff
@@ -95,6 +95,17 @@ sealed trait ConfigCursor {
   def asMap: Either[ConfigReaderFailures, Map[String, ConfigCursor]] =
     asObjectCursor.right.map(_.map)
 
+  final def atPath(key: String, moreKeys: String*): Either[ConfigReaderFailures, ConfigCursor] = {
+    val allKeysLeadingUpToLast :+ lastKey = key +: moreKeys
+    val finalObject = allKeysLeadingUpToLast
+      .foldLeft(this.asObjectCursor) {
```

Comment:
Now that the method is in `ConfigCursor`, you don't need to extract and treat `lastKey` in a separate way. You can simplify this expression by folding over `key +: moreKeys` with an initial value of `this`.